src/js/actions/DialogActions.js

```diff
@@ -1,72 +1,84 @@
+var Util = require("../helpers/Util");
 var AppDispatcher = require("../AppDispatcher");
 var DialogEvents = require("../events/DialogEvents");
+var DialogTypes = require("../constants/DialogTypes");
+var dialogScheme = require("../stores/schemes/dialogScheme");
 
 var DialogActions = {
-  alert: function (message, dismissButtonLabel = "OK") {
-    const dialogId = Symbol(message);
-    AppDispatcher.dispatchNext({
-      actionType: DialogEvents.ALERT_SHOW,
-      dialogId: dialogId,
-      dismissButtonLabel: dismissButtonLabel,
-      message: message
-    });
-    return dialogId;
-  },
-  alertDismiss: function (dialogId) {
-    AppDispatcher.dispatchNext({
-      actionType: DialogEvents.ALERT_DISMISS,
-      dialogId: dialogId
-    });
-  },
-  confirm: function (message, successButtonLabel = "OK") {
-    const dialogId = Symbol(message);
+  dialog: function (data) {
```

Comment:
@aldipower I've investigated the DialogActions proposal and would really love to have such a simple API, but as you mentioned it's not compatible with our Flux structure. :disappointed:  I'm going to make the showDialog method private, even though I liked the current design better but enforcing  consistent usage is probably a good idea. Thanks for your input!